doc/ocpp_201_status.md

```diff
@@ -1160,7 +1160,7 @@ This document contains the status of which OCPP 2.0.1 numbered requirements have
 | K01.FR.20 | :white_check_mark: |  |
```

Comment:
I know it is not specific to this change but can you uncheck FR.20 and add a Remark?